### PR TITLE
add Loading component with correct branch name

### PR DESCRIPTION
**Pull request by Juett**

### CODE REVIEW

File: js/components/Loading/Loading.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    return (
+      <View style={styles.loadingContainer}>
+        <LinearGradient
+          start={{ x: 0, y: 0 }}
+          end={{ x: 0, y: 0.8 }}
+          colors={["#BADDD6", "#FAE4D9"]}
+          style={[StyleSheet.absoluteFill, { height: "100%", width: "100%" }]}
+        />
+        <Animated.View
+          style={{
```
Comment (siddharthparmar7) [inline js/components/Loading/Loading.js:49]: move your styles to styles.js file

File: js/components/Loading/Loading.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      inputRange: [0, 1],
+      outputRange: ["0deg", "360deg"]
+    });
+    return (
+      <View style={styles.loadingContainer}>
+        <LinearGradient
+          start={{ x: 0, y: 0 }}
+          end={{ x: 0, y: 0.8 }}
+          colors={["#BADDD6", "#FAE4D9"]}
+          style={[StyleSheet.absoluteFill, { height: "100%", width: "100%" }]}
```
Comment (siddharthparmar7) [inline js/components/Loading/Loading.js]: move your styles and use Dimensions not %.

File: js/components/Loading/Loading.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        />
+        <Animated.View
+          style={{
+            position: "absolute",
+            top: height / 3.75,
+            left: width / 4.6,
+            transform: [{ rotate: spin }]
+          }}
+        >
+          <Image
```
Comment (siddharthparmar7) [inline js/components/Loading/Loading.js]: use Activity Indicator and colour prop instead. https://facebook.github.io/react-native/docs/activityindicator

File: js/screens/Journal/Journal.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 import PropTypes from "prop-types";
-import { Text, View } from "react-native";
+import Loading from "../../components/Loading";
 
 const Journal = () => {
   return (
-    <View>
-      <Text>Journal</Text>
-    </View>
+    <Fragment>
```
Comment (siddharthparmar7) [inline js/screens/Journal/Journal.js]: remove Fragment Wrapper. there are not more than 2 component in there. Just return your Loading component.

File: js/screens/Journal/Journal.js [hunk 5]
```diff
@@ -1,12 +1,12 @@
-import React from "react";
+import React, {Fragment} from "react";
 import PropTypes from "prop-types";
```
Comment (siddharthparmar7) [inline js/screens/Journal/Journal.js]: remove unnecessary imports.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please squash your commits. Make sure you are following conventions while creating a branch name, no block letter in the branch name. Please make the requested changes. Thank you.

File: js/components/Loading/Loading.js (hunk 3 above)
Comment (Juett) [reply js/components/Loading/Loading.js]: ActivityIndicator would have been easier but it doesn't match the design.

File: js/screens/Journal/Journal.js (hunk 4 above)
Comment (Juett) [reply js/screens/Journal/Journal.js]: done

File: js/screens/Journal/Journal.js (hunk 5 above)
Comment (Juett) [reply js/screens/Journal/Journal.js]: done

File: js/components/Loading/Loading.js (hunk 2 above)
Comment (Juett) [reply js/components/Loading/Loading.js]: done

File: js/components/Loading/Loading.js (hunk 3 above)
Comment (siddharthparmar7) [reply js/components/Loading/Loading.js]: show me

File: js/components/Loading/Loading.js (hunk 2 above)
Comment (siddharthparmar7) [inline js/components/Loading/Loading.js]: not done here. Please make the changes.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**